test/integration/scheduler_test.go

```diff
@@ -275,57 +275,6 @@ func DoTestUnschedulableNodes(t *testing.T, restClient *client.Client, nodeStore
 	}
 }
 
-func BenchmarkScheduling(b *testing.B) {
-	framework.DeleteAllEtcdKeys()
-
-	var m *master.Master
-	s := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		m.Handler.ServeHTTP(w, req)
-	}))
-	defer s.Close()
-
-	masterConfig := framework.NewIntegrationTestMasterConfig()
-	m = master.New(masterConfig)
-
-	c := client.NewOrDie(&client.Config{
-		Host:         s.URL,
-		GroupVersion: testapi.Default.GroupVersion(),
-		QPS:          5000.0,
-		Burst:        5000,
-	})
-
-	schedulerConfigFactory := factory.NewConfigFactory(c, nil)
-	schedulerConfig, err := schedulerConfigFactory.Create()
-	if err != nil {
-		b.Fatalf("Couldn't create scheduler config: %v", err)
-	}
-	eventBroadcaster := record.NewBroadcaster()
-	schedulerConfig.Recorder = eventBroadcaster.NewRecorder(api.EventSource{Component: "scheduler"})
-	eventBroadcaster.StartRecordingToSink(c.Events(""))
-	scheduler.New(schedulerConfig).Run()
-
-	defer close(schedulerConfig.StopEverything)
-
-	makeNNodes(c, 1000)
-	N := b.N
-	b.ResetTimer()
-	makeNPods(c, N)
-	for {
-		objs := schedulerConfigFactory.ScheduledPodLister.Store.List()
-		if len(objs) >= N {
-			fmt.Printf("%v pods scheduled.\n", len(objs))
-			/* // To prove that this actually works:
-			for _, o := range objs {
-				fmt.Printf("%s\n", o.(*api.Pod).Spec.NodeName)
-			}
-			*/
-			break
-		}
-		time.Sleep(time.Millisecond)
-	}
-	b.StopTimer()
-}
-
 func makeNNodes(c client.Interface, N int) {
```

Comment:
If `func BenchmarkScheduling(b *testing.B)` is removed, then `func makeNNodes` and `func makeNPods` are not useful any more, maybe they should be removed as well. 

PS: package "sync" should be removed as well should `makeNPods`and `makeNNodes` be deleted.